package.json

```diff
@@ -9,7 +9,8 @@
     "test": "NODE_ENV=test && jest --watch",
     "test:ci": "jest --ci --passWithNoTests",
     "local": "docker-compose up -d && npx prisma studio",
-    "stop-local": "docker-compose down"
+    "stop-local": "docker-compose down",
+    "postinstall": "npx prisma generate"
```

Comment:
I added this due to Vercel issue where it caches dependencies which gives prisma client problems. link: https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/vercel-caching-issue#:~:text=Prisma%20ORM%20uses%20a%20postinstall,sync%20with%20your%20database%20schema.